pkg/metadata/labels.go

```diff
@@ -41,10 +41,17 @@ const (
 	InstanceLabel          = LabelKROPrefix + "instance-name"
 	InstanceNamespaceLabel = LabelKROPrefix + "instance-namespace"
 
+	// Parent Identifier Labels. These are set by the instance reconciler that creates/updates the resource.
 	ResourceGraphDefinitionIDLabel        = LabelKROPrefix + "resource-graph-definition-id"
```

Comment:
Do we want this to be `created-by`? Possibly we can keep these (back-compat) and add new ones and remove the old later.

Reply:
made this PR track the proposal #639. Once we merge that proposal i will change the implementation here. Closing all comments for now.